accessibility-checker-engine/help-v4/en-US/table_layout_linearized.html

```diff
@@ -70,8 +72,8 @@ <h3 id="ruleMessage"></h3>
 
 ### Who does this affect?
 
- * People using a screen reader, including blind, low vision and neurodivergent people
- * People who use a braille display
+ * People using a screen reader, including blind, low vision, and neurodivergent people
+ * People who use a Braille display
```

Comment:
Lowercase correct here
```suggestion
 * People who use a braille display
```